index.html

```diff
@@ -194,46 +354,51 @@ <h1 class="about-me">About me</h1>
                 don’t hesitate to contact me.</p>
             <button class="button-3" type="submit">Get my resume</button>
         </section>
+        
         <!-- background img-2 -->
+        
         <section  class="div-img">
-        <img src="Illustration-2.png" alt="" class="img-3">
-           </section>
-           
+            <img src="images/background-desktop-2.png" alt="" class="background-desktop-2">
+            <img src="Illustration-2.png" alt="" class="img-3">
```

Comment:
I believe the image in the about section is causing your website to break on different screen sizes. Please consider making this image a background image as it is decorative and not part of the main content. This way, it wont break your site👍